services/subscriptions/service.go

```diff
@@ -1,8 +1,9 @@
 package subscriptions
 
 import (
+	"net/http"
```

Comment:
why did we switch to net/http from echo?

Reply:
I think its because we are just using the http library methods, is that right?

Reply:
Yeah. Just the constants like `http.MethodPost`. We're still using Echo at higher levels.